package.json

```diff
@@ -1,31 +1,41 @@
 {
-  "name": "skyux-split-view",
-  "version": "1.0.0",
-  "description": "Single-page-application for skyux-split-view",
+  "name": "@skyux/split-view",
+  "version": "3.0.0-alpha",
```

Comment:
Just confirming, this will be `3.0.0-alpha.0`?

Reply:
Yep, I'll fix that.

Reply:
Done.